src/cascade/executor/epiviz_runner.py

```diff
@@ -72,8 +76,13 @@ def add_mortality_data(model_context, execution_context, sex_id):
     MATHLOG.debug(f"Creating a set of mtspecific observations from IHME CSMR database.")
     MATHLOG.debug("Assigning standard error from measured upper and lower.")
 
+    if execution_context.parameters.tier == 3:
+        raw_csmr = normalize_mortality_data(get_cause_specific_mortality_data(execution_context))
```

Comment:
get_csmr and get_cause_specific_mortality are the same meaning. Maybe one should be get_frozen and get_nonfrozen, as with asdr below.

Reply:
Yeah